pkgs/tools/misc/oggvideotools/default.nix

```diff
@@ -43,5 +43,6 @@ stdenv.mkDerivation rec {
     homepage = "http://www.streamnik.de/oggvideotools.html";
     license = licenses.gpl2Only;
     maintainers = with maintainers; [ SuperSandro2000 ];
+    broken = stdenv.isDarwin;
```

Comment:
Can we add a very short comment with the reason?

Reply:
This is done